Find a functionally equivalent code:android.graphics.Paint.setShader:COMMENT
Method Modifier: public      
Comment:/**
 * Set or clear the shader object.
 * <p />
 * Pass null to clear any previous shader.
 * As a convenience, the parameter passed is also returned.
 *
 * @param shader May be null. the new shader to be installed in the paint
 * @return       shader
 */

@@ -1,7 +1,7 @@
 {
-    int shaderNative = 0;
+    long shaderNative = 0;
     if (shader != null)
-        shaderNative = shader.native_instance;
+        shaderNative = shader.getNativeInstance();
     native_setShader(mNativePaint, shaderNative);
     mShader = shader;
     return shader;

